cli/src/main/scala/mdoc/internal/cli/Settings.scala

```diff
@@ -294,14 +303,20 @@ object Settings { // extends MetaconfigScalametaImplicits with Decoders with Set
     generic.deriveDecoder[Settings](base)
   }
 
-  def fromCliArgs(args: List[String], base: Settings): Configured[Settings] = {
+  def fromCliArgs(args: List[String], workingDirectory: Path): Configured[Settings] = {
     Conf
       .parseCliArgs[Settings](args)
       .andThen(conf => {
+        val base =
+          Settings.default(
+            AbsolutePath(workingDirectory),
+            conf.get[String]("propertyFileName").getOrElse("mdoc.properties")
```

Comment:
I could be I'm missing something, but as far as I can tell, this would require the constant to be moved outside of the `Settings` class - it would essentially become a `global` variable in `mdoc.internal.cli`

Is this what you intend? It would not be my choice.